code/modules/mob/living/ventcrawling.dm

```diff
@@ -1,11 +1,11 @@
+// VENTCRAWLING
+// Handles the entrance and exit on ventcrawling
+/mob/living/proc/handle_ventcrawl(obj/machinery/atmospherics/components/A)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names